Modules/_functoolsmodule.c

```diff
@@ -1789,6 +1795,10 @@ _functools_exec(PyObject *module)
     // lru_list_elem is used only in _lru_cache_wrapper.
     // So we don't expose it in module namespace.
 
+    if (PyModule_Add(module, "_initial_missing", _PyObject_New(&PyBaseObject_Type)) < 0) {
```

Comment:
I guess this is ok, but I would prefer to not use the private API where it doesn't help anything:
```suggestion
    if (PyModule_Add(module, "_initial_missing", PyObject_New(PyObject, &PyBaseObject_Type)) < 0) {
```

Reply:
I have made this change locally (didn't apply this because the changed line was too long), will push it with other changes. Thanks.

Reply:
Perhaps, a more pedantic version is:
```suggestion
    PyObject *missing = PyObject_New(PyObject, &PyBaseObject_Type);
    if (!missing || PyModule_Add(module, "_initial_missing", missing) < 0) {
        Py_XDECREF(missing);
```

Reply:
Probably not--the benefit of `PyModule_Add` is that you don't need to do something like that, because it comes with a `NULL` check and steals the reference anyway. That looks like the kind of thing that would get used with `PyModule_AddObjectRef`